Makefile.common

```diff
@@ -57,18 +57,27 @@ endif
 # articles and only books can have appendices) and then strip out the
 # bits we don't want/need
 #
+# XXX Sequence of sed commands should be replaced with a single XSLT
+#
 %.xml:  %.txt
-	$(AM_V_ASCII)$(ASCIIDOC) -b docbook -d book -o $@ $<
+if IS_ASCIIDOC
+	$(AM_V_ASCII)$(ASCIIDOC_CONV) -b docbook -d book -o $@ $<
+else
+	$(AM_V_ASCII)$(ASCIIDOC_CONV) -b docbook45 -d book -o $@ $<
+endif
 	$(AM_V_at)sed -i 's///' $@
-	$(AM_V_at)sed -i 's/
//' $@                 # Fix line endings
-	$(AM_V_at)sed -i 's/\ lang="en"//' $@        # Never specify a language in the chapters
-	$(AM_V_at)sed -i 's/simpara/para/g' $@       # publican doesn't correctly render footnotes with simpara
-	$(AM_V_at)sed -i 's/.*<date>.*//g' $@	       # Remove dangling tag
-	$(AM_V_at)sed -i 's/.*preface>//g' $@        # Remove preface elements
-	$(AM_V_at)sed -i 's:<title></title>::g' $@   # Remove empty title
-	$(AM_V_at)sed -i 's/chapter/section/g' $@    # Chapters become sections, so that books can become chapters
-	$(AM_V_at)sed -i 's/<.*bookinfo.*>//g' $@    # Strip out bookinfo, we don't need it
-	-grep -qis "<appendix" $@ && sed -i 's/.*book>//' $@         # We just want the appendix tag
-	-grep -vqis "<appendix" $@ && sed -i 's/book>/chapter>/g' $@ # Rename to chapter
+	$(AM_V_at)sed -i 's/
+	$(AM_V_at)sed -i 's/
//' $@                # Fix line endings
```

Comment:
Are the above two lines a github display issue or a code issue?

Reply:
Oops, sorry for this, might be related to me copying something from
garbled output.

Reply:
(so yes, root cause might have been a display issue, but rather on my terminal)

Reply:
Opened https://github.com/isaacs/github/issues/1329 as a reminder
about this confusing behaviour.